src/main/java/catan/domain/model/user/UserBean.java

```diff
@@ -25,13 +27,21 @@
     @Column(name = "GUEST", nullable = false)
     private boolean guest;
 
+    @Column(name = "LANGUAGE")
+    private String language;
+
+    @Column(name = "COUNTRY")
+    private String country;
+
     public UserBean() {
     }
 
     public UserBean(String username, String password, boolean guest) {
         this.username = username;
         this.password = password;
         this.guest = guest;
+        this.language = "en";
+        this.country = "US";
```

Comment:
why do you hardcode these values in constructor, I would move it somewhere higher, so that I can easily change it if needed